backend/src/data/migrations/20210812090556_add_cover_id_to_podcast.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+
+const TABLE_NAME = 'podcasts';
+const COLUMN_NAME = 'cover_id';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.integer(COLUMN_NAME).defaultTo(null);
```

Comment:
what about relation?
it should be relation with image table

don't forget about model, types and etc